test/test-script.js

```diff
@@ -0,0 +1,256 @@
+const chai = require('chai'),  
+    spies = require('chai-spies');
+    expect = chai.expect, 
+    assert = chai.assert,
+    should = chai.should();
+
+chai.use(spies);
+
+//MOCK GLOBAL STORAGE
+function storageMock() {
+    var storage = {};
+
+    return {
+      setItem: function(key, value) {
+        storage[key] = value || '';
+      },
+      getItem: function(key) {
+        return key in storage ? storage[key] : null;
+      },
+      removeItem: function(key) {
+        delete storage[key];
+      },
+      get length() {
+        return Object.keys(storage).length;
+      },
+      key: function(i) {
+        var keys = Object.keys(storage);
+        return keys[i] || null;
+      }
+    };
+  }
+
+// MOCK CHROME before calling the tested code
+const chrome = require('chrome-mock');
+global.chrome = chrome;
+global.handleMessage=  {};
+localStorage = storageMock();
+
+//calling the tested code
+const testedScript = require('../script.js');
+
+
+
```

Comment:
Delete empty lines ?

Reply:
I have the same remark as here : https://github.com/supertanuki/Carbonalyser/pull/23#discussion_r361156686